packages/website/pages/docs/concepts/proposing.mdx

```diff
@@ -60,4 +71,22 @@ A transaction is valid if and only if:
 2. The transaction's signature is valid (rule #2 in Ethereum yellow paper).
 3. The transaction's gas limit is no smaller than the intrinsic gas $K_{TxMinGasLimit}$ (rule #5 in the Ethereum yellow paper).
 
-If any of these fails, a throwaway block will instead be created on L2 whose first transaction is `invalidateBlock`, and this will prove the proposed block as invalid on TaikoL1.
+If any of these fails, an empty block will be created on L2 (with only the anchor tx).
+
+## Anchor transaction
+
+The anchor transaction is a way for the protocol to make use of the programmability of the EVM (which we already need to be able to proof) to enforce certain protocol behavior. We can add additional tasks to anchor transactions to enrich Taiko’s functionalities by writing standard smart contract code (instead of requiring more complicated changes to Taiko’s ZK-EVM and node subsystems).
+
+The anchor transaction is required to be the first transaction in a Taiko block (which is important to make the block deterministic). The anchor transaction is currently used as follows:
+1. Persisting `l1Height` $C_a$ and `l1Hash` $C_h$, data inherited from L1, to the storage trie. These values can be used by bridges to validate cross-chain messages.
```

Comment:
also `l1SignalRoot` and `parentGasUsed`?
<img width="890" alt="image" src="https://github.com/taikoxyz/taiko-mono/assets/104078303/403de831-6129-4186-9bdf-1a1690da86eb">